src/__tests__/starWarsSchema.js

```diff
@@ -164,7 +167,7 @@ const humanType = new GraphQLObjectType({
       type: new GraphQLList(characterInterface),
       description: 'The friends of the human, or an empty list if they ' +
                    'have none.',
-      resolve: human => getFriends(human),
+      resolve: (human: Human): Array<Promise<Character>> => getFriends(human),
```

Comment:
Maybe it is necessary, the input and output datatype should be a part of  schema's design. A explicit declare give the resolver a constraint.(Makes it clear to other people, what data should be in and what data should be out).